i18n/pt-BR/pop_desktop_widget.ftl

```diff
@@ -56,7 +56,7 @@ gis-panel-title = Configure a Barra Superior
 hot-corner = Hot Corner
 hot-corner-description = Ativar Hot Corner do canto superior esquerdo para exibir as Áreas de Trabalho
 
-multi-monitor-behavior = Comportamento nos Monitores
+multi-monitor-behavior = Comportamento para Multi-Monitores
```

Comment:
This change is nearly un-doing what another PR tweaked: https://github.com/pop-os/desktop-widget/pull/41/files 

I'm inclined to say including `Multi-` is correct since this is the name of a feature (not necessarily translatable), but I could see it not being appropriate if we're trying to translate the feature name. @cappp Do you have any thoughts on this before we merge it? Can you or @ComicShrimp explain why it should be one way or the other? (Just trying to get it figured out so there's not continuous back-and-forth.)

Reply:
I think switching to "Multi" to keep the feature name is good! Maybe it's better to recognize.

Reply:
I think that `Multi` is more accurate. For me, "Comportamento nos Monitores" could include just one monitor, while `Multi-` is more specific.

I think that `Comportmento nos Monitores` sound better to the ears, but `Comportamento para Multi-Monitores` seems more accurate in this case.

Reply:
Thank you both!